lib/Translation/ExportVerilog/ExportVerilog.cpp

```diff
@@ -2296,7 +2303,7 @@ LogicalResult StmtEmitter::visitStmt(InstanceOp op) {
     emitLocationInfoAndNewLine(ops);
 
     // Emit the port's name.
-    indent();
+    indent() << prefix;
```

Comment:
I'm not very familiar with the code but this looks like it could be moved to line 2311 to avoid double-commenting zero-width ports?

Reply:
I think I want the double comment.  in the normal output it would have one comment, and I want to look like a block comment applied to the output.